stdlib/process.jou

```diff
@@ -9,6 +9,12 @@ declare system(command: byte*) -> int
 # Exit the current process.
 declare exit(status: int) -> noreturn  # Clean exit. Status 0 is success, other status (usually 1) means fail.
 declare abort() -> noreturn  # Hard exit. Similar to a crash.
+declare execl(path: byte*, arg: byte*, ...) -> int # Execute the executable file with argument list args, replacing the current process.
+
+declare access(pathname: byte*, mode: int) -> int # Use the real uid/gid to test for access to a path.
```

Comment:
This function has nothing to do with processes. Maybe we should make a separate `paths.jou` which would be similar to `pathlib` and `os.path` in Python?

Reply:
Maybe os.jou?
Jou miss a os module

Reply:
Jou doesn't need to mirror Python perfectly, but maybe `os.jou` would be good because Python users expect to find some things in `os`.